packages/neuron-ui/src/components/Send/hooks.ts

```diff
@@ -100,6 +103,8 @@ const updateTransactionWith = (generator: typeof generateTx | typeof generateSen
         setErrorMessage(err.message)
         return undefined
       })
+  } catch {
```

Comment:
Are there any potential side effects if this error is silenced?

Reply:
It has the same functionality before the update. The `catch` here is similar to
```js
const onSubmit = () => {
  if (disabled) {
    return
  }
}
```